### ISSUE TITLE
Accessibility Type model

### ISSUE THREAD

**Issue by cmgrulke** [Chemical-Curation/chemcurator_django#179]
**As a** product owner
**I want** to be able to store an accessibility type as a controlled vocabulary of the generic accessibility a list should have when reading
**So that** I can inform the data hub of content that should be restricted as to whom may view it

A accessibility type consists of:
- Name = String (Less than 50 character, url safe, unique, required field)
- Label = String (Less than 100 characters, unique, required field)
- Short Description = String (Less than 500 characters, required field)
- Long Description = TEXT (required field)

**Acceptance Criteria**
A test to show when a accessibility type is created it works properly...
